.github/workflows/android-cd.yml

```
@@ -0,0 +1,68 @@
+        with:
+          java-version: '21'
+          distribution: 'temurin'
+          cache: gradle
+          
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+
+      - name: Add Local Properties
+        run:
+          echo base.url=\"$_BASE_URL\" >> ./local.properties
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)